framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -429,6 +418,17 @@ protected boolean isContainedBy(
         }
     }
 
+    private boolean ignoreUninferred(AnnotatedTypeMirror inside) {
```

Comment:
I would maybe add the `TypeArgument` to the method name and use `type` instead of `inside`, to avoid confusion with the parameter names used in the calls of this method.